patches/third_party-widevine-cdm-widevine.gni.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
+index 58f073ca562ca40c3165f4dd0f650dc467b43d5c..8668af33c10b49df982ec4eb92cfaefe9302feff 100644
+--- a/third_party/widevine/cdm/widevine.gni
++++ b/third_party/widevine/cdm/widevine.gni
+@@ -20,10 +20,10 @@ declare_args() {
+ if (is_chromeos && !is_chromeos_device) {
+   enable_widevine = false
+ }
+-
+ # Widevine CDM is available as a library CDM on the following platforms and
+ # architectures. Notably on Android library CDM is not used and Widevine is
+ # supported via Android MediaDrm API.
++declare_args() { # Brave adds declare_args here in anticipation that upstream will do the same.
```

Comment:
as Brian mentioned, no comments needed, so just `declare_args() {`.

Reply:
Resolved in 07e9aeebff86e2f79ba0b39aa06bdb6291f39659.